paychmgr/simple.go

```diff
@@ -98,7 +98,7 @@ func (ca *channelAccessor) processNextQueueItem() {
 	}
 
 	head := ca.fundsReqQueue[0]
-	res := ca.processTask(head.ctx, head.from, head.to, head.amt, head.onComplete)
```

Comment:
Is there any reason to keep `fundsReq.onComplete`?

Reply:
Yes we actually call it on line 116. We need it because we queue tasks up and then execute them later, so we need to be able to call back to tell the task when it has been executed.

Reply:
(I had put it in a different place in an earlier refactor and forgot to remove it until now)